modules/hal_nordic/Kconfig

```diff
@@ -7,6 +7,10 @@ config ZEPHYR_HAL_NORDIC_MODULE
 config HAS_NORDIC_DRIVERS
 	bool
 
+config MPSL
+	bool
+	# Dummy kconfig to allow compiling outside of NCS
```

Comment:
Please change the help text.

A pure Zephyr user would have no understanding of what NCS means, or why a dummy var is needed.
Please give a proper description of the variable.